authors/apps/authentication/renderers.py

```diff
@@ -15,11 +16,17 @@ def render(self, data, media_type=None, renderer_context=None):
         errors = data.get('errors', None)
 
 
+
         if errors:
             """
             As mentioned about, we will let the default JSONRenderer handle
             rendering errors.
             """
+
+        if errors is not None:
```

Comment:
@mikenthiwa using is not None is quite repetitive given that `None` is falsy in python so using `if errors` should work.